### PR TITLE
Adding manual control

### PR DESCRIPTION
**Pull request by CodeReaper**
I have created the feature I am missing in match as described in #74. Summed up, a method to add certificates, keys and profiles to a match repo without having access to the Dev portal they were downloaded from.

Besides the regular feedback, I would especially appreciate feedback on my handling of arguments, since I have added a number of global arguments. These arguments should ideally only be available for the sub command `manual`, but I am unsure how to accomplish that.

### CODE REVIEW

**Comment by CodeReaper** [comment]
I have attempted to get the same violations as hound and failures from travis locally, but I am not having much luck doing that. In order to make appropriate corrections to pass the the checks, I need a little help in making it fail :)

File: README.md [hunk 1]
```diff
@@ -297,6 +297,24 @@ match change_password
 
 You'll be asked for the new password on all your machines on the next run.
 
+### Manual control
+
+If you for any reason do not have access to the Dev portal, you can still save your certificates and profiles in match.
+
+To save your certificate, run.
+```
+match manual -p path/to/key.p12 -c path/to/my.mobileprovision -r https://github.com/fastlane/certificates -a tools.fastlane.app -y appstore
```
Comment (KrauseFx) [inline README.md]: Do you think it's possible to implement something like `match import` which just detects all those values from the local keychain and imports everything?

Comment (CodeReaper) [inline README.md]: You might be able to find the known identities in the keychain, but matching certificates up against the correct mobileprovision may prove troublesome... or at least I do not know how to do that.

**Comment by kcharwood** [comment]
I agree this would be _very_ valuable to me. I do not want to revoke my existing enterprise cert/profile, but would like to be able to distribute it to other CI machines this way.

**Comment by CodeReaper** [comment]
@KrauseFx Having been thru the pull request with the verify_build action yesterday, should I try to make this pull request also consist of a single commit?

**Comment by mfurtak** [comment]
Hi @CodeReaper - we're about to get moving on a big reorganization of the fastlane repos. All of the tools, including `match`, are going to change locations to be in the new all-encompassing `fastlane` repo. As part of that, we're unfortunately not going to be able to bring pull requests over with us.

Apologies, but I'm going to close this for now since it is not in state where we can get it in. Sorry about that! 😣

**Comment by kcharwood** [comment]
Please link the new PR to this one!

Thanks.

**Pull request closed without merging**